Cleanup namespace that were moved to ATen accidentally

bwasti opened the pull request:
Summary: torch::jit shouldn't live in aten

Differential Revision: D10389502

ezyang commented on torch/csrc/jit/interned_strings.h:
```diff
@@ -1 +1,11 @@
+
+namespace torch {
+namespace jit {
+namespace prim = ::c10::prim;
```
OOC, is there a reason to prefer this over 'namespace prim { using namespace c10::prim }'?

bwasti replied:
I'll do that

bwasti committed:
Cleanup namespace that were moved to ATen accidentally (#12680)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/12680

torch::jit shouldn't live in aten

Reviewed By: ezyang

Differential Revision: D10389502

fbshipit-source-id: 27b86d99db809743b1fbaf503b3ddc0a24372a18
